.github/scripts/utils.py

```diff
@@ -1,6 +1,8 @@
 import os
 import json
 import glob
+import urllib
```

Comment:
is url lib being used? didn't see anything on the changes below

Reply:
Good spot... That was used by a function I had added here, but then I moved the function to `add_footer.py`, to be closer to related code - meaning the new import was no longer needed here....

I've pushed the fix.